ext/standard/filestat.c

```diff
@@ -911,7 +905,19 @@ PHPAPI void php_stat(const char *filename, size_t filename_length, int type, zva
 		RETURN_TRUE; /* the false case was done earlier */
 	case FS_LSTAT:
 		/* FALLTHROUGH */
-	case FS_STAT:
+	case FS_STAT: {
+		char *stat_sb_names[] = {
+			"dev", "ino", "mode", "nlink", "uid", "gid", "rdev",
+			"size", "atime", "mtime", "ctime", "blksize", "blocks"
+		};
+		zval stat_dev, stat_ino, stat_mode, stat_nlink, stat_uid, stat_gid, stat_rdev,
+			stat_size, stat_atime, stat_mtime, stat_ctime, stat_blksize, stat_blocks;
+		zval *stat_sb_addresses[] = {
+			&stat_dev, &stat_ino, &stat_mode, &stat_nlink, &stat_uid, &stat_gid, &stat_rdev,
+			&stat_size, &stat_atime, &stat_mtime, &stat_ctime, &stat_blksize, &stat_blocks
+		};
+		size_t i, size_stat_sb = sizeof(stat_sb_addresses) / sizeof(*stat_sb_addresses);
```

Comment:
`I386_DEBUG_ZTS` build failed. Maybe is it because of this?

Reply:
That failure looks intermittent, and not related to this PR.